### PR TITLE
[SYCLomatic] Fixes for DPCT1013 diagnostics

### PR DESCRIPTION
**Pull request by vladimir-polin**
then->than typo fixed and "precision" changed to "accuracy" which is more relevant to this particular diagnostics.

Signed-off-by: Vladimir Polin Vladimir.Polin@intel.com

### CODE REVIEW

File: clang/lib/DPCT/Diagnostics.inc [hunk 1]
```diff
@@ -39,8 +39,8 @@ DEF_WARNING(TRNA_WARNING_OVERLOADED_API_FOUND, 1011, "The tool detected overload
...
 DEF_COMMENT(TIME_MEASUREMENT_FOUND, 1012, "Detected kernel execution time measurement pattern and generated an initial code for time measurements in SYCL. You can change the way time is measured depending on your goals.")
-DEF_WARNING(ROUNDING_MODE_UNSUPPORTED, 1013, "The rounding mode could not be specified and the generated code may have different precision then the original code. Verify the correctness. SYCL math built-ins rounding mode is aligned with OpenCL C 1.2 standard.")
-DEF_COMMENT(ROUNDING_MODE_UNSUPPORTED, 1013, "The rounding mode could not be specified and the generated code may have different precision then the original code. Verify the correctness. SYCL math built-ins rounding mode is aligned with OpenCL C 1.2 standard.")
+DEF_WARNING(ROUNDING_MODE_UNSUPPORTED, 1013, "The rounding mode could not be specified and the generated code may have different accuracy than the original code. Verify the correctness. SYCL math built-ins rounding mode is aligned with OpenCL C 1.2 standard.")
```
Comment (zhiweij1) [inline clang/lib/DPCT/Diagnostics.inc:42]: I still prefer "precision" to "accuracy". @zhimingwang36 what's your opinion?

Comment (vladimir-polin) [reply clang/lib/DPCT/Diagnostics.inc:42]: Look at https://en.wikipedia.org/wiki/Rounding article
"Precision" term is mostly for double precision, single precision, integer, etc. 
"Accuracy" term or "Accurate rounding" is used for rounding modes themselves 
So unless there is unexpected DP->SP precision loss covered by this diagnostics I suggest using "accuracy".

Comment (zhimingwang36) [reply clang/lib/DPCT/Diagnostics.inc:42]: LGTM

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**